### ISSUE TITLE
Unable to rename files

### ISSUE THREAD

**Issue by NjaalBendixen** [quodlibet/quodlibet#2205]
Dear Quod Libet people

I am new to this forum, so please forgive me if I miss the etiquette.

I am a tango dj and I use Quod Libet as my music organizer, because it is the only music library program that allows me to organize the music the way that I need to organize it!!!  Thank you for your great work so far!!!!

Here is the issue:

When I try to rename files (either to rename them only or to move them around the filesystem) then I get this error message:

_________________________________________

Unable to rename file

Renaming Burundi__Drummers_Of_Burundi,_The__Instrumental__Live_at_real_world______Cede__The_Drummers_of_Burundi____01__.flac to /home/njaal/Music/MUSICAquodlibet/MUSICAflac/Burundi/Drummers_Of_Burundi,_The/Instrumentales/Burundi/Sin_ano/Burundi__Drummers_Of_Burundi,_The__Instrumental__Live_at_real_world______Cede__The_Drummers_of_Burundi____01__.flac failed. Possibly the target file already exists, or you do not have permission to make the new file or remove the old one.

______________________________

This happens everytime also with Ex Falso. I have been able to do this before with the same file path and file name.

OS: Linux mint 18, xfce
QL: 3.8.0

Thank you very much 

Njaal Bendixen

**Comment by lazka** [comment]
Thanks for the report.

Could you either start "exfalso --debug" or "quodlibet --debug" in a terminal and try to renamed like before. You should see an error message when doing the rename. Could you provide terminal output printed after you try to rename?

(We should make it possible to show the traceback in the error dialog... with some details expander)

**Comment by NjaalBendixen** [comment]
Hi

Here is the information requested. I am not sure which form you want it. I hope this is OK. If not please let me know what to do.



Here is a screen shot of
![screenshot_2017-01-25_16-06-13](https://cloud.githubusercontent.com/assets/25297408/22298916/21ebd474-e31a-11e6-9bba-faa02429a335.png)
 the error message that came up after I did "quodlibet --debug" in a terminal:

_______________

Here is the printout of the terminal after doing "quodlibet --debug":


njal@Ceibo ~ $ quodlibet --debug
D: 0.217: _init._init_gettext: LANGUAGE: None
D: 0.218: _init._init_gettext: LANG: 'en_GB.UTF-8'
D: 0.315: _init._init_gettext: Translations loaded: '/usr/share/locale/en_GB/LC_MESSAGES/quodlibet.mo'
D: 0.768: formats._misc.init: MimeType=application/x-flac;audio/aac;audio/aiff;audio/midi;audio/mp3;audio/mp4;audio/mpeg;audio/mpeg4;audio/mpg;audio/ogg;audio/vorbis;audio/wav;audio/wave;audio/x-aac;audio/x-aiff;audio/x-flac;audio/x-m4a;audio/x-midi;audio/x-mp3;audio/x-mpc;audio/x-mpeg;audio/x-ms-wma;audio/x-ms-wmv;audio/x-musepack;audio/x-oggflac;audio/x-speex;audio/x-tta;audio/x-wav;audio/x-wavpack;audio/x-wma;video/ogg;video/x-ms-asf;video/x-theora;video/x-wmv;
D: 0.771: formats._misc.init: Supported formats: aac, aiff, midi, mod, monkeysaudio, mp3, mp4, mpc, spc, trueaudio, vgm, wav, wavpack, wma, xiph
D: 2.103: _init._init_gtk_debug: Initializing debugging extensions
D: 2.476: qltk.add_signal_watch: Register Python signal handler: 'SIGHUP'
D: 2.476: qltk.add_signal_watch: Register Python signal handler: 'SIGINT'
D: 2.476: qltk.add_signal_watch: Register Python signal handler: 'SIGTERM'
D: 2.477: qltk.add_signal_watch: Register GLib signal handler: 'SIGHUP'
D: 2.477: qltk.add_signal_watch: Register GLib signal handler: 'SIGINT'
D: 2.477: qltk.add_signal_watch: Register GLib signal handler: 'SIGTERM'
D: 3.512: main.main: Initializing main library (~/.quodlibet/songs)
D: 3.515: SongFileLibrary.__init__: Initializing SongFileLibrary "main".
D: 3.515: SongFileLibrary.__init__: Using pickling persistence for library "main"
D: 3.515: SongFileLibrary.load: Loading contents of '/home/njal/.quodlibet/songs'.
D: 5.753: SongFileLibrary.load(<libraries.SongFileLibrary object at 0x7f1a041e5690 (quodlibet+library+libraries+SongFileLibrary at 0x2d9ec40)>): Done loading contents of '/home/njal/.quodlibet/songs'.
D: 6.384: devices.init: Initialising device backend.
D: 6.386: devices.init: Trying 'UDisks2'
D: 6.412: devices.init: Device backend initialised.
D: 7.341: PlaylistsBrowser.load_pattern: Loading pattern from /home/njal/.quodlibet/playlist_pattern
D: 7.342: PlaylistsBrowser.load_pattern: Couldn't load pattern for PlaylistsBrowser ([Errno 2] No such file or directory: '/home/njal/.quodlibet/playlist_pattern'), using default.
D: 7.357: AlbumList.load_pattern: Loading pattern from /home/njal/.quodlibet/album_pattern
D: 7.358: AlbumList.load_pattern: Couldn't load pattern for AlbumList ([Errno 2] No such file or directory: '/home/njal/.quodlibet/album_pattern'), using default.
D: 7.366: CoverGrid.load_pattern: Loading pattern from /home/njal/.quodlibet/album_pattern
D: 7.373: CoverGrid.load_pattern: Couldn't load pattern for CoverGrid ([Errno 2] No such file or directory: '/home/njal/.quodlibet/album_pattern'), using default.
D: 7.373: SongFileLibrary.__init__: Initializing SongFileLibrary "filesystem".
D: 7.373: SongFileLibrary.__init__: Using pickling persistence for library "filesystem"
D: 7.449: UDisks2Manager._try_build_device: Found device: '/dev/sda1'
D: 7.816: UDisks2Manager._try_build_device: '/dev/sda1' not a media player
D: 7.825: _main.init_plugins: Starting plugin manager
D: 7.826: _main.init_plugins: Scanning folders: ['/usr/lib/python2.7/dist-packages/quodlibet/ext/editing', '/usr/lib/python2.7/dist-packages/quodlibet/ext/events', '/usr/lib/python2.7/dist-packages/quodlibet/ext/playorder', '/usr/lib/python2.7/dist-packages/quodlibet/ext/songsmenu', '/usr/lib/python2.7/dist-packages/quodlibet/ext/playlist', '/usr/lib/python2.7/dist-packages/quodlibet/ext/gstreamer', '/usr/lib/python2.7/dist-packages/quodlibet/ext/covers', '/usr/lib/python2.7/dist-packages/quodlibet/ext/query', '/home/njal/.quodlibet/plugins']
D: 7.826: plugins.__restore: Restoring plugins: 1
D: 7.827: PluginManager.rescan: Rescanning..
D: 7.827: ModuleScanner.rescan: Rescanning..
D: 9.170: ModuleScanner.rescan: Rescanning done: 82 added, 0 removed, 4 error(s)
D: 9.201: PluginManager.rescan: Rescanning done.
D: 9.207: PluginManager.register_handler: Registering handler: 'EditTagsPluginHandler'
D: 9.207: PluginManager.register_handler: Registering handler: 'RenameFilesPluginHandler'
D: 9.208: PluginManager.register_handler: Registering handler: 'TagsFromPathPluginHandler'
D: 9.209: PluginManager.register_handler: Registering handler: 'GStreamerPlayer'
D: 9.226: PluginManager.register_handler: Registering handler: 'SongsMenuPluginHandler'
D: 9.227: PluginManager.register_handler: Registering handler: 'CoverPluginHandler'
D: 9.229: PluginManager.register_handler: Registering handler: 'PlaylistPluginHandler'
D: 9.231: PluginManager.register_handler: Registering handler: 'QueryPluginHandler'
D: 9.970: PluginManager.register_handler: Registering handler: 'PluggableOrders'
D: 9.988: PluginManager.register_handler: Registering handler: 'PluggableOrders'
D: 10.009: qltk.playorder.__compose_order: Updating PlaylistModel order to <In Order>
D: 10.183: GStreamerPlayer.go_to: Going to None
D: 10.187: PlaylistMux.go_to: Told to go to None
D: 10.187: QueueModel.go_to: Told to go to None
D: 10.187: PlaylistModel.go_to: Told to go to None
D: 10.188: SongLibrary.__init__: Using pickling persistence for library "iradio-remote"
D: 10.188: SongLibrary.load: Loading contents of '/home/njal/.quodlibet/stations_all'.
D: 10.917: SongLibrary.load(<libraries.SongLibrary object at 0x7f19f2b40500 (quodlibet+library+libraries+SongLibrary at 0x2dc7200)>): Done loading contents of '/home/njal/.quodlibet/stations_all'.
D: 10.923: SongLibrary.__init__: Using pickling persistence for library "iradio"
D: 10.924: SongLibrary.load: Loading contents of '/home/njal/.quodlibet/stations'.
W: 10.924: library.libraries._load_items: Couldn't load library file from: '/home/njal/.quodlibet/stations'
D: 10.924: SongLibrary.load: Done loading contents of '/home/njal/.quodlibet/stations'.
D: 11.418: PlaylistModel.set: Filling view model with 47 songs.
D: 11.427: GStreamerPlayer.go_to: Going to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.431: PlaylistMux.go_to: Told to go to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.434: QueueModel.go_to: Told to go to None
D: 11.434: PlaylistModel.go_to: Told to go to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.437: GStreamerPlayer._end: End song
D: 11.469: util.cover.manager.failure: Didn't get cover from FilesystemCover: This source is incapable of fetching covers
D: 11.477: util.cover.manager.failure: Didn't get cover from EmbeddedCover: This source is incapable of fetching covers
D: 11.479: GStreamerPlayer._end: Next song
D: 12.456: qltk.window.__restore_state: Restore state
D: 12.457: qltk.window.__restore_position: Restore position
D: 12.457: qltk.window.__restore_size: Restore size
D: 12.459: PluginManager.register_handler: Registering handler: 'EventPluginHandler'
D: 12.463: PluginManager.enable: Enable 'Equalizer'
D: 15.279: mmkeys.find_active_backend: Trying to find a mmkeys backend
D: 15.355: qltk.session.init: Connecting with the gnome session manager failed
D: 15.356: _main.run: Entering quodlibet.main
D: 16.095: player.gstbe.player.__message: Recalculate latency
D: 17.895: _init._override_exceptions: Enabling custom exception handler.
D: 18.400: qltk.completion.__build_model: Updating tag model for whole library
D: 19.111: qltk.completion.__build_model: Done updating tag model for whole library

______________________________________


________________________________________
Here is the dumpfile
dumpfile name: Dump_20170125_160221.txt

=== SYSTEM INFORMATION:
Quod Libet 3.8.0
Mutagen 1.36
Python 2.7.11+ (default, Apr 17 2016, 14:00:29) 
[GCC 5.3.1 20160413] linux2
Platform Linux-4.4.0-21-generic-x86_64-with-LinuxMint-18-sarah
=== STACK TRACE

Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/quodlibet/_main.py", line 283, in quit_gtk
    before_quit()

  File "/usr/lib/python2.7/dist-packages/quodlibet/main.py", line 202, in before_quit
    print_d("Shutting down player device %r." % player.version_info)

  File "/usr/lib/python2.7/dist-packages/quodlibet/util/dprint.py", line 282, in print_d
    _print_message(string, context, True, "D", "green", "debug")

  File "/usr/lib/python2.7/dist-packages/quodlibet/util/dprint.py", line 217, in _print_message
    print_(string, file=file_, flush=True)

  File "/usr/lib/python2.7/dist-packages/quodlibet/senf/_print.py", line 59, in print_
    _print_unix(objects, sep, end, file, flush)

  File "/usr/lib/python2.7/dist-packages/quodlibet/senf/_print.py", line 103, in _print_unix
    file.write(data)

IOError: [Errno 5] Input/output error

=== LOG:
D: 2.477: qltk.add_signal_watch: Register GLib signal handler: 'SIGINT'
D: 2.477: qltk.add_signal_watch: Register GLib signal handler: 'SIGTERM'
D: 3.512: main.main: Initializing main library (~/.quodlibet/songs)
D: 3.515: SongFileLibrary.__init__: Initializing SongFileLibrary "main".
D: 3.515: SongFileLibrary.__init__: Using pickling persistence for library "main"
D: 3.515: SongFileLibrary.load: Loading contents of '/home/njal/.quodlibet/songs'.
D: 5.753: SongFileLibrary.load(<libraries.SongFileLibrary object at 0x7f1a041e5690 (quodlibet+library+libraries+SongFileLibrary at 0x2d9ec40)>): Done loading contents of '/home/njal/.quodlibet/songs'.
D: 6.384: devices.init: Initialising device backend.
D: 6.386: devices.init: Trying 'UDisks2'
D: 6.412: devices.init: Device backend initialised.
D: 7.341: PlaylistsBrowser.load_pattern: Loading pattern from /home/njal/.quodlibet/playlist_pattern
D: 7.342: PlaylistsBrowser.load_pattern: Couldn't load pattern for PlaylistsBrowser ([Errno 2] No such file or directory: '/home/njal/.quodlibet/playlist_pattern'), using default.
D: 7.357: AlbumList.load_pattern: Loading pattern from /home/njal/.quodlibet/album_pattern
D: 7.358: AlbumList.load_pattern: Couldn't load pattern for AlbumList ([Errno 2] No such file or directory: '/home/njal/.quodlibet/album_pattern'), using default.
D: 7.366: CoverGrid.load_pattern: Loading pattern from /home/njal/.quodlibet/album_pattern
D: 7.373: CoverGrid.load_pattern: Couldn't load pattern for CoverGrid ([Errno 2] No such file or directory: '/home/njal/.quodlibet/album_pattern'), using default.
D: 7.373: SongFileLibrary.__init__: Initializing SongFileLibrary "filesystem".
D: 7.373: SongFileLibrary.__init__: Using pickling persistence for library "filesystem"
D: 7.449: UDisks2Manager._try_build_device: Found device: '/dev/sda1'
D: 7.816: UDisks2Manager._try_build_device: '/dev/sda1' not a media player
D: 7.825: _main.init_plugins: Starting plugin manager
D: 7.826: _main.init_plugins: Scanning folders: ['/usr/lib/python2.7/dist-packages/quodlibet/ext/editing', '/usr/lib/python2.7/dist-packages/quodlibet/ext/events', '/usr/lib/python2.7/dist-packages/quodlibet/ext/playorder', '/usr/lib/python2.7/dist-packages/quodlibet/ext/songsmenu', '/usr/lib/python2.7/dist-packages/quodlibet/ext/playlist', '/usr/lib/python2.7/dist-packages/quodlibet/ext/gstreamer', '/usr/lib/python2.7/dist-packages/quodlibet/ext/covers', '/usr/lib/python2.7/dist-packages/quodlibet/ext/query', '/home/njal/.quodlibet/plugins']
D: 7.826: plugins.__restore: Restoring plugins: 1
D: 7.827: PluginManager.rescan: Rescanning..
D: 7.827: ModuleScanner.rescan: Rescanning..
D: 9.170: ModuleScanner.rescan: Rescanning done: 82 added, 0 removed, 4 error(s)
D: 9.201: PluginManager.rescan: Rescanning done.
D: 9.207: PluginManager.register_handler: Registering handler: 'EditTagsPluginHandler'
D: 9.207: PluginManager.register_handler: Registering handler: 'RenameFilesPluginHandler'
D: 9.208: PluginManager.register_handler: Registering handler: 'TagsFromPathPluginHandler'
D: 9.209: PluginManager.register_handler: Registering handler: 'GStreamerPlayer'
D: 9.226: PluginManager.register_handler: Registering handler: 'SongsMenuPluginHandler'
D: 9.227: PluginManager.register_handler: Registering handler: 'CoverPluginHandler'
D: 9.229: PluginManager.register_handler: Registering handler: 'PlaylistPluginHandler'
D: 9.231: PluginManager.register_handler: Registering handler: 'QueryPluginHandler'
D: 9.970: PluginManager.register_handler: Registering handler: 'PluggableOrders'
D: 9.988: PluginManager.register_handler: Registering handler: 'PluggableOrders'
D: 10.009: qltk.playorder.__compose_order: Updating PlaylistModel order to <In Order>
D: 10.183: GStreamerPlayer.go_to: Going to None
D: 10.187: PlaylistMux.go_to: Told to go to None
D: 10.187: QueueModel.go_to: Told to go to None
D: 10.187: PlaylistModel.go_to: Told to go to None
D: 10.188: SongLibrary.__init__: Using pickling persistence for library "iradio-remote"
D: 10.188: SongLibrary.load: Loading contents of '/home/njal/.quodlibet/stations_all'.
D: 10.917: SongLibrary.load(<libraries.SongLibrary object at 0x7f19f2b40500 (quodlibet+library+libraries+SongLibrary at 0x2dc7200)>): Done loading contents of '/home/njal/.quodlibet/stations_all'.
D: 10.923: SongLibrary.__init__: Using pickling persistence for library "iradio"
D: 10.924: SongLibrary.load: Loading contents of '/home/njal/.quodlibet/stations'.
W: 10.924: library.libraries._load_items: Couldn't load library file from: '/home/njal/.quodlibet/stations'
D: 10.924: SongLibrary.load: Done loading contents of '/home/njal/.quodlibet/stations'.
D: 11.418: PlaylistModel.set: Filling view model with 47 songs.
D: 11.427: GStreamerPlayer.go_to: Going to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.431: PlaylistMux.go_to: Told to go to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.434: QueueModel.go_to: Told to go to None
D: 11.434: PlaylistModel.go_to: Told to go to u'http://streaming202.radionomy.com:80/feelingtheblues'
D: 11.437: GStreamerPlayer._end: End song
D: 11.469: util.cover.manager.failure: Didn't get cover from FilesystemCover: This source is incapable of fetching covers
D: 11.477: util.cover.manager.failure: Didn't get cover from EmbeddedCover: This source is incapable of fetching covers
D: 11.479: GStreamerPlayer._end: Next song
D: 12.456: qltk.window.__restore_state: Restore state
D: 12.457: qltk.window.__restore_position: Restore position
D: 12.457: qltk.window.__restore_size: Restore size
D: 12.459: PluginManager.register_handler: Registering handler: 'EventPluginHandler'
D: 12.463: PluginManager.enable: Enable 'Equalizer'
D: 15.279: mmkeys.find_active_backend: Trying to find a mmkeys backend
D: 15.355: qltk.session.init: Connecting with the gnome session manager failed
D: 15.356: _main.run: Entering quodlibet.main
D: 16.095: player.gstbe.player.__message: Recalculate latency
D: 17.895: _init._override_exceptions: Enabling custom exception handler.
D: 18.400: qltk.completion.__build_model: Updating tag model for whole library
D: 19.111: qltk.completion.__build_model: Done updating tag model for whole library
D: 44.232: qltk.handler: GLib signal handler activated: SIGHUP
D: 44.236: qltk.window.__restore_state: Restore state
D: 44.237: qltk.window.__restore_position: Restore position
D: 44.238: qltk.window.__restore_size: Restore size
D: 44.241: main.before_quit: Saving active browser state



_____________________________________________


______________________________--

Here is the mini dumpfile
Minidump file name: MiniDump_20170125_160221.txt



=== SYSTEM INFORMATION:
Quod Libet 3.8.0
Mutagen 1.36
Python 2.7.11+ (default, Apr 17 2016, 14:00:29) 
[GCC 5.3.1 20160413] linux2
Platform Linux-4.4.0-21-generic-x86_64-with-LinuxMint-18-sarah
=== STACK TRACE

Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/quodlibet/_main.py", line 283, in quit_gtk
    before_quit()

  File "/usr/lib/python2.7/dist-packages/quodlibet/main.py", line 202, in before_quit
    print_d("Shutting down player device %r." % player.version_info)

  File "/usr/lib/python2.7/dist-packages/quodlibet/util/dprint.py", line 282, in print_d
    _print_message(string, context, True, "D", "green", "debug")

  File "/usr/lib/python2.7/dist-packages/quodlibet/util/dprint.py", line 217, in _print_message
    print_(string, file=file_, flush=True)

  File "/usr/lib/python2.7/dist-packages/quodlibet/senf/_print.py", line 59, in print_
    _print_unix(objects, sep, end, file, flush)

  File "/usr/lib/python2.7/dist-packages/quodlibet/senf/_print.py", line 103, in _print_unix
    file.write(data)

IOError: [Errno 5] Input/output error



______________________________


Here is the printout from "quodlibet --debug" after I tried to rename:


njal@Ceibo ~ $ quodlibet --debug
D: 0.063: _init._init_gettext: LANGUAGE: None
D: 0.063: _init._init_gettext: LANG: 'en_GB.UTF-8'
D: 0.103: _init._init_gettext: Translations loaded: '/usr/share/locale/en_GB/LC_MESSAGES/quodlibet.mo'
D: 0.226: formats._misc.init: MimeType=application/x-flac;audio/aac;audio/aiff;audio/midi;audio/mp3;audio/mp4;audio/mpeg;audio/mpeg4;audio/mpg;audio/ogg;audio/vorbis;audio/wav;audio/wave;audio/x-aac;audio/x-aiff;audio/x-flac;audio/x-m4a;audio/x-midi;audio/x-mp3;audio/x-mpc;audio/x-mpeg;audio/x-ms-wma;audio/x-ms-wmv;audio/x-musepack;audio/x-oggflac;audio/x-speex;audio/x-tta;audio/x-wav;audio/x-wavpack;audio/x-wma;video/ogg;video/x-ms-asf;video/x-theora;video/x-wmv;
D: 0.227: formats._misc.init: Supported formats: aac, aiff, midi, mod, monkeysaudio, mp3, mp4, mpc, spc, trueaudio, vgm, wav, wavpack, wma, xiph
njal@Ceibo ~ $ 




_________________________________-


Hope this is what you wanted. Please let me know if I should have done something differently.





Hope this is what you are looking for.

**Comment by lazka** [comment]
Hm, ok the error message you get is because you closed the terminal before quodlibet.

Did you try to rename files names? I can't see anything in the logs.

**Comment by NjaalBendixen** [comment]
Yes, I did try to rename files after, get the same error message. see screenshot:

![screenshot](https://cloud.githubusercontent.com/assets/25297408/22307416/c8a2f854-e33a-11e6-8814-5a33134cbf40.png)

**Comment by NjaalBendixen** [comment]
One thing I tried was to uninstall QL and install it again. Somehow some of the library information was retained. This is wierd. If I unsinstall a program, then I expect the program to completely disappear. Uninstalling is something that people do when they want to star over. If you need to retain previous information after reinstalling, then it should somehow be possible to export the library file or something like that. If the program retains itself after re-installation, then it is impossible to "just start over".

**Comment by declension** [comment]
@NjaalBendixen many users want the opposite actually, but anyway. If you want to start from fresh, please delete (or move) `~/.quodlibet`.

Back to the ticket though, I notice that those filenames look (by eye) identical other than capitalisations. This isn't a Windows mounted filesystem is it?

Also, _does_ that file exist already?

**Comment by NjaalBendixen** [comment]
The file system was created by Qoud Libet running on Linux. I have used this file system with several versions of linux, and with several different computers.



OK, I understand that people want continuity from one install to another. If I can start form scratch by deleting the ~/.quodlibet file, then that is fine by me. A very easy solution.

This is exactly what I did. I uninstalled QL, then I deleted the ~/.quodlibet file. Then I installed QL again and recreated the library and tried to rename again.


I tried to rename to following file:

~/Music/MUSICAquodlibet/XXX/Buffalo/MUSICAflac/bad_sound/De_Angelis,_Alfredo/Instrumentales/Tango/1945/Tango__De_Angelis,_Alfredo__Instrumental__El_pial__1945-1-9__1945__iPod_de_Mimi,_Angelis,_Alfredo_de,_desde_cede_From_Argentina_to_the_world_00821838343929____01__.flac



It did not work. This is what happened; and the destination file does definitely not exist!

First I tried to rename the file with the formula that I always use:

/home/njaal/Music/MUSICAquodlibet/MUSICAflac/<biblioteca|<biblioteca>|Otra>/<orquesta>/<voz|<voz>|Instrumentales>/<genero>/<ano|<ano>|Sin_ano>/<genero>; <orquesta>; <voz|<voz>|Instrumental>; <title>; <fecha>; <ano>; <fuente>; <discnumber>; <tracknumber>; <dublicado>


I ticked the following boxes:

Replace spaces with underscores
Strip |Windows-incomptible characters
Strip diacritical marks
Strip non-ASCII characters


Then I got this error message:

Unable to rename file

Renaming Tango__De_Angelis,_Alfredo__Instrumental__El_pial__1945-1-9__1945__iPod_de_Mimi,_Angelis,_Alfredo_de,_desde_cede_From_Argentina_to_the_world_00821838343929____01__.flac to /home/njaal/Music/MUSICAquodlibet/MUSICAflac/bad_sound/De_Angelis,_Alfredo/Instrumentales/Tango/1945/Tango__De_Angelis,_Alfredo__Instrumental__El_pial__1945-1-9__1945__iPod_de_Mimi,_Angelis,_Alfredo_de,_desde_cede_From_Argentina_to_the_world_00821838343929____01__.flac failed. Possibly the target file already exists, or you do not have permission to make the new file or remove the old one.


Then I tried with a simpler rename formula:
 
/home/njaal/Music/MUSICAquodlibet/<title>


I got the same error message.


What should I do next? I really need to sort this issue out. Should I try to reinstall linux?

**Comment by NjaalBendixen** [comment]
Sorry everyone. 

This is all my own fault, I have been the silly one.

I apologize for having waisted everyones time.

This is what happened:

Ny name is Njål, so the home folder used to be home/njaal/
in my last reinstall of linux this got changed to home/njal/

When I tried to rename I copied and pasted from the archives that I used to use before my last reintall, which had the extra a in it.

Now everything works just fine.

Thank you all for your considerations


Kind regards

Njaal Bendixen
